keyboards/synthlabs/solo/config.h

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 Aaron Hong (@hongaaronc)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x534E // "SN"
+#define PRODUCT_ID      0x3031 // "01"
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Synth Labs
+#define PRODUCT         Solo
+
+#define ENCODERS 1
+#define ENCODER_RESOLUTION 4.5
+#define ENCODERS_PAD_A { B2 }
+#define ENCODERS_PAD_B { B3 }
+#define ENCODERS_CW_KEY { { 2, 1 } }
+#define ENCODERS_CCW_KEY { { 0, 1 } }
+
+/* key matrix size */
+#define MATRIX_ROWS 3
+#define MATRIX_COLS 7
+
+/* Pins D0 and D2 are actually unused, but are included in DIRECT_PINS to fill space in the keyboard matrix to hold CW and CCW keycodes */
```

Comment:
Couldn't you use `NO_PIN` for these, instead?

Reply:
Ah, yep! That makes way more sense - thanks for pointing that out. Changed as of 5c9de2621058c1a50e4fd867e77157331cc6acf5 .

Reply:
Actually, I ended up removing this declaration entirely as per your other suggestion in https://github.com/qmk/qmk_firmware/pull/16103#discussion_r810872075

Replaced these dummy values with `null` in `info.json` as of e1ab7730d4836b1c4e61193470b54ccba087da63